pkg/apis/apis/v1alpha1/types.go

```diff
@@ -177,6 +197,15 @@ const (
 	// BindingResourceDeleteSuccess is a condition for APIBinding that indicates the resources relating this binding are deleted
 	// successfully when the APIBinding is deleting
 	BindingResourceDeleteSuccess conditionsv1alpha1.ConditionType = "BindingResourceDeleteSuccess"
+
+	// AcceptedPermissionClaimInvalid is a condition for APIBinding that indicates that there is an invalid accepted permission claim.
+	AcceptedPermissionClaimValid conditionsv1alpha1.ConditionType = "AcceptedPermissionClaimValid"
```

Comment:
is "valid" the right term? `AcceptedPermissionClaimsComplete` maybe?

Reply:
+1